### PR TITLE
Opaque integer resources documentation.

### PR DESCRIPTION
**Pull request by aveshagarwal**
@derekwaynecarr please cc others as needed.

### CODE REVIEW

File: dev_guide/compute_resources.adoc [hunk 1]
```diff
@@ -281,6 +281,89 @@ $ oc run nginx --image=nginx --limits=cpu=200m,memory=400Mi --requests=cpu=100m,
...
+
+Opaque integer resources allow cluster operators to advertise new node-level
+resources that would be otherwise unknown to the system. Users can consume
+these resources in pod specs just like CPU and memory. The scheduler takes
+care of the resource accounting so that no more than the available amount is
+simultaneously allocated to pods.
+
+[NOTE]
+====
+Opaque integer resources are Alpha currently, and only resource accounting is
```
Comment (ahardin-rh) [inline dev_guide/compute_resources.adoc]: Is Alpha the same as Technology Preview?

**Comment by ahardin-rh** [comment]
Thanks! Is this for 3.4?

**Comment by ahardin-rh** [comment]
@aveshagarwal @derekwaynecarr Is this for 3.5?

**Comment by aveshagarwal** [comment]
@ahardin-rh yes.

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/compute_resources.adoc#dev-guide-compute-resources[Quotas and Limit Ranges]
|Added a new section on xref:../dev_guide/compute_resources.adoc#opaque-integer-resources[Opaque Integer Resources].
%

**Pull request merged**